app/models/npq_application.rb

```diff
@@ -148,10 +148,17 @@ def previously_funded?
       .where.not(id:)
       .where(npq_course: npq_course.rebranded_alternative_courses)
       .where(eligible_for_funding: true)
+      .where(funded_place: [nil, true])
```

Comment:
Does this have to be a nullable attribute, can we not default it to `true`? it makes it harder to follow/less readable if we need to check for `nil` and `true` and I imagine its something that could trip us up in the future?

Reply:
discussed in person: good point, it would make life much easier if we did default to true, however old values will be surfaced in the API and will need to be empty. Also we require providers to supply this value so we cannot default it to true and need to distinguish values submitted by them vs defaulted.